test-js/unit/pentaho/visual/base/View.Spec.js

```diff
@@ -0,0 +1,57 @@
+define([
+  "pentaho/visual/base/View"
+], function(View) {
+  "use strict";
+
+  describe("pentaho/visual/base/View", function() {
+    var element, dataSpec;
+
+    beforeEach(function() {
+      element = document.createElement("div");
+      var data = {};
+      dataSpec = {v: data};
+    });
+
+    describe("the constructor ", function() {
+
+      it("should throw if not invoked with two arguments", function() {
+        expect(function() {
+          return new View();
+        }).toThrowError(/required/);
```

Comment:
How do I know which of the required errors is being thrown? Why not follow the pattern we've been using? I know you dislike it, but unless you propose a better alternative, I'd prefer staying with what we've got.